tests/ModelTest.php

```diff
@@ -1048,12 +1048,17 @@ public function testNumericFieldName(): void
 
     public function testCreateOrFirst()
     {
+        Carbon::setTestNow('2010-06-22');
+        $createdAt = Carbon::now()->getTimestamp();
         $user1 = User::createOrFirst(['email' => 'john.doe@example.com']);
 
         $this->assertSame('john.doe@example.com', $user1->email);
         $this->assertNull($user1->name);
         $this->assertTrue($user1->wasRecentlyCreated);
+        $this->assertEquals($createdAt, $user1->created_at->getTimestamp());
```

Comment:
Should this use a comparison to assert that the `created_at` and `updated_at` fields are greater than or equal to the previously calculated current time?

I don't see anything in the PR that would guarantee the entire test completes within a single timestamp.

Reply:
`Carbon::setTestNow` freeze the clock, so that the date point that is used to populate `create_at` is the on that is set.